### ISSUE TITLE
System.NullReferenceException upon WebView2.Dispose

### ISSUE THREAD

**Issue by mikemeinz** [MicrosoftEdge/WebView2Feedback#442]
**Description**
This worked fine using 0.9.579-prerelease with Target CPU **AnyCPU** and **Prefer 32 Bit**. This version of my program is built with Target CPU X64 using 0.9.628-prerelease. 

My program (running in Debug Mode) is getting ready to shutdown. Within the **FormClosing** event, I call **WebView2.Stop** and then call **WebView2.Dispose**. The exception occurs within **Microsoft.Web.WebView2.WinForms.WebView2.OnVisibleChanged**.

From **Copy Details** of the exception MessageBox:
```
System.NullReferenceException
  HResult=0x80004003
  Message=Object reference not set to an instance of an object.
  Source=Microsoft.Web.WebView2.WinForms
  StackTrace:
   at Microsoft.Web.WebView2.WinForms.WebView2.OnVisibleChanged(EventArgs e)
   at System.Windows.Forms.Control.WmShowWindow(Message& m)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.DebuggableCallback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

I commented out **WebView.Dispose** and program runs OK.  Maybe calling Dispose is not necessary?

**Version**
SDK: 0.9.628-prerelease
Runtime:  Version: 86.0.622.15 MicrosoftEdgeWebView2RuntimeInstallerX64
Framework: WinForms .NET 4.8
OS: Windows 10 Version 1909 (OS Build 18363.1016)

**Repro Steps**
`WebView2.Dispose`

**Screenshots**
The exception MessageBox
![image](https://user-images.githubusercontent.com/13871412/93271500-6bd22f00-f781-11ea-92e1-35cb0b37f42d.png)

Visual Studio Exception Settings
![image](https://user-images.githubusercontent.com/13871412/93272781-50b4ee80-f784-11ea-8b1f-0717e790aa50.png)

**Additional context**

[AB#29391058](https://microsoft.visualstudio.com/90b2a23c-cab8-4e7c-90e7-a977f32c1f5d/_workitems/edit/29391058)

**Comment by cgeier** [comment]
I noticed the same issue in C#. 

```
System.NullReferenceException
  HResult=0x80004003
  Message=Object reference not set to an instance of an object.
  Source=Microsoft.Web.WebView2.WinForms
  StackTrace:
   at Microsoft.Web.WebView2.WinForms.WebView2.OnVisibleChanged(EventArgs e)
   at System.Windows.Forms.Control.WmShowWindow(Message& m)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.DebuggableCallback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```



![Form_Closing_NullReferenceException](https://user-images.githubusercontent.com/34412039/93285136-270ac000-f7a2-11ea-8127-8d8dbcddd695.JPG)

**Form1_FormClosing:**
```
private void Form1_FormClosing(object sender, FormClosingEventArgs e)
{
    Debug.WriteLine("Info: Form1_FormClosing");
    Debug.WriteLine("webView21.CoreWebView2.BrowserProcessId 1: " + webView21.CoreWebView2.BrowserProcessId);
    Debug.WriteLine("webView21.IsDisposed: " + webView21.IsDisposed + " webView21.BrowserProcessId: ");

    if (webView21 != null && !webView21.IsDisposed)
    {
        Debug.WriteLine("webView21.Visible: " + webView21.Visible);
        Debug.WriteLine("webView21.CoreWebView2.BrowserProcessId - in IF: " + webView21.CoreWebView2.BrowserProcessId);

        webView21.Dispose();   // <---System.NullReferenceException: 'Object reference not set to an instance of an object'
    } 
}

```

**Output window:**
```
Info: Form1_FormClosing
webView21.CoreWebView2.BrowserProcessId 1: 8732
webView21.IsDisposed: False webView21.BrowserProcessId: 
webView21.Visible: True
webView21.CoreWebView2.BrowserProcessId - in IF: 8732
webView21_VisibleChanged
An unhandled exception of type 'System.NullReferenceException' occurred in Microsoft.Web.WebView2.WinForms.dll
Object reference not set to an instance of an object.
```

Inside `if (webView21 != null && !webView21.IsDisposed)` the Debug statements output values, so webView21 isn't null.

**Comment by ukandrewc** [comment]
@mikemeinz Getting this too, workaround for me is: **Webview2.visible = False** before disposing.

**Comment by champnic** [comment]
This should now be fixed in the latest 1.0.790-prerelease SDK package. Thanks!

**Comment by ralluin** [comment]
Buil x64
